extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -84,6 +92,14 @@ class AmpStickyAd extends AMP.BaseElement {
     }
   }
 
+  removeOnHorizontalListener_() {
+    if (this.bindedOnHorizontalScroll_) {
+      this.element.removeEventListener('scroll',
+          this.bindedOnHorizontalScroll_);
+      this.bindedOnHorizontalScroll_ = null;
```

Comment:
Is there a reason to throw away the listener? (Not unbind it -- we definitely should do that. But why throw away the reference to the bound function)